src/states/node.rs

```diff
@@ -731,17 +700,6 @@ impl Node {
         Ok(false)
     }
 
-    fn dynamic_quorum_size(&self) -> usize {
```

Comment:
Some quorum is still needed. This is all provisional? I wonder if it would have been better to replace this with `network_size / 2 + 1` for now?